<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SateMainWindow</class>
 <widget class="QMainWindow" name="SateMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1704</width>
    <height>955</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SATE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sate_resources.qrc">
    <normaloff>:/img/icons/sate.png</normaloff>:/img/icons/sate.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1704</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDarkMode"/>
    <addaction name="actionLightMode"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionOpenObject"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDarkMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionLightMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light Mode</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="actionConnectWs">
   <property name="text">
    <string>Connect websocket...</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit Sate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenObject">
   <property name="text">
    <string>Open Object...</string>
   </property>
   <property name="toolTip">
    <string>Deserialize object from XML or JSON file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="sate_resources.qrc"/>
 </resources>
 <connections/>
</ui>
